src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs

```diff
@@ -83,9 +94,11 @@ public InternalPersonnelController()
 
             #endregion
 
+            var command = new GetDepartmentPersonnel(fullDepartmentString, query)
+                .WithTimeline(shouldExpandTimeline, timelineStart, timelineEnd);
+            command.IncludeSubdepartments = includeSubdepartments;
```

Comment:
Should follow the same pattern with fluent. 
`.IncludeSubDepartments(includeSubDepartments)`